tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/Configs.scala

```diff
@@ -539,6 +542,32 @@ object Configs {
                                 config.getString("service"),
                                 config.getString("admin"),
                                 options)
+      case SourceCategory.HBASE =>
+        val fields: ListBuffer[String] = new ListBuffer[String]
+        for (field <- config.getStringList("fields").asScala) {
+          fields.append(field)
+        }
+
+        if (config.hasPath("vertex")) {
+          fields.append(config.getString("vertex"))
+        }
+        if (config.hasPath("source.field")) {
+          fields.append(config.getString("source.field"))
+        }
+        if (config.hasPath("target.field")) {
+          fields.append(config.getString("target.field"))
+        }
+
+        HBaseSourceConfigEntry(
+          SourceCategory.HBASE,
+          getOrElse(config, "sentence", ""),
```

Comment:
Actually, HBase do not support sentence?

Reply:
Yes, sentence parameter for HBaseSourceConfigEntry should be removed.